portfolio/src/main/webapp/about_me.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```

Comment:
Anything you want to put in this file? Since there is no content in the main div.

Reply:
Yes this were I was going to put my resume, student organization I am part of, my specific interest in computer science and maybe some side projects when they are done.